depends/packages/qt.mk

```diff
@@ -281,12 +281,12 @@ define $(package)_build_cmds
 endef
 
 define $(package)_stage_cmds
-  $(MAKE) -C qtbase/src INSTALL_ROOT=$($(package)_staging_dir) $(addsuffix -install_subtargets,$(addprefix sub-,$($(package)_qt_libs))) && \
-  $(MAKE) -C qttools/src/linguist INSTALL_ROOT=$($(package)_staging_dir) $(addsuffix -install_subtargets,$(addprefix sub-,$($(package)_linguist_tools))) && \
+  $(MAKE) -C qtbase INSTALL_ROOT=$($(package)_staging_dir) install && \
```

Comment:
Can you explain this change? It's been ages since I've messed with the qt build, but IIRC this was done to keep the installed libs to a minimum. Now we just install everything?

Reply:
Otherwise, the CMake-driven configuration fails. I cannot point at the exact error as I did that change a year ago and just don't remember the exact problem.

Reply:
Ok. Sounds like maybe it just added some additional cmake targets added to our list? Otherwise I believe this means `$(package)_qt_libs` is now unused.

 Mind adding a comment?

"TODO: Investigate whether specific targets can be used here to minimize the amount of files/components installed.".

Reply:
> Mind adding a comment?
> 
> "TODO: Investigate whether specific targets can be used here to minimize the amount of files/components installed.".

Added.